folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -83,6 +85,14 @@
   private final int maxSetLength;
 
   private final AtomicReference<String> disconnectReason = new AtomicReference<>(null);
+  private static final ThreadFactory THREAD_FACTORY =
+      new DefaultThreadFactory(DefaultRawMemcacheClient.class, true);
+  private static final Supplier<EventLoopGroup> DEFAULT_EVENT_LOOP_GROUP =
```

Comment:
is it bad to reuse the same EventLoopGroup by default?

I tried shutting down the EventLoopGroup in the disconnect-method, but that stalled infinitely

Reply:
this might help: https://stackoverflow.com/questions/47134860/is-it-a-good-idea-to-use-the-same-netty-eventloopgroup-for-both-serverbootstrap

(I was curious about the same question)